app/components/go_component.html.erb

```diff
@@ -0,0 +1,20 @@
+<div class="go">
+  <div class="captures">
+    <div>
+      <% @captures["black"].times do %>
+      <span class="black"></span>
+      <% end %>
+    </div>
+    <div>
+      <% @captures["white"].times do %>
+      <span class="white"></span>
+      <% end %>
+    </div>
+  </div>
+
+  <div class="board <%= @current %>">
+    <% @board.each_with_index do |v,i| %>
+    <%= button_tag "", class: v.to_s, data: { motion: "place", index: i.to_s } %>
```

Comment:
Just a note that since this gets converted into HTML, the numbers are automatically converted into strings, and the nil board vertices are empty as well.